src/routes/docs/configuring/replace-file-explorer.svx

```diff
@@ -1,24 +1,11 @@
-# Replacing File Explorer with Files (Unsupported)
+# Replacing File Explorer with Files (Experimental)
 
-*This guide involves modifying the registry, make sure to create a backup beforehand and proceed at your own risk.
-Please keep in mind that this method is unsupported and may not work for everyone.*
+*This setting modifies the system registry, make sure to create a backup beforehand and proceed at your own risk.
+Please keep in mind that these changes are not reverted when removing the app so make sure to turn this setting off before removing the app from your device.*
```

Comment:
Out of curiosity, does installing Files again make it possible to revert this setting?

Reply:
Yes

Reply:
Good to know, thanks!